docs/src/environments/_index.md

```diff
@@ -1,118 +1,165 @@
 description: Learn what environments on Platform.sh are and how to take advantage of them.
 ---
 
-On Platform.sh, an environment is a logically separate instance of an app or group of apps
-with all of the services needed to run the app.
+A Platform.sh environment contains one instance of an app (or [group of apps](../create-apps/multi-app.md))
+with all the services needed for it to run.
 
-You can think of an environment as a complete working website
-that's related to but safely isolated from others in the project.
-So you can run tests and review a complete working copy without worrying about damaging anything in production.
-
-Each project includes multiple environments,
+Each project can include multiple environments,
 often divided into [environment types](../administration/users.md#environment-types).
 If you have a live site, you have at least a production environment.
-You may also have additional environments for development, testing, staging, review, and so on.
-
-New environments can be created by branching existing environments using the [command line interface (CLI)](/administration/cli/_index.md),
-or in the [Console](../administration/web/_index.md).
-Each created environment is an exact replica of its parent environment.
-This means new environments have all of the data and services from the parent
-(like databases, network storage, queues, routing).
+You can think of other environments as copies of your live site,
+where you can run tests without worrying about damaging anything in production.
+Once you have completed your tests,
+you can merge your changes so that they're instantly and seamlessly deployed to production.
 
+You may create environments for development, testing, staging, review, and so on.
 To organize your environments, you can create [hierarchical relationships](#hierarchy).
 
-An environment is tied to a Git branch and can be created on demand.
-With Bitbucket and GitHub integrations you can even get a "development server" automatically for each and every pull request.
+## Create environments
+
+To create new environments, you can:
+
+-   push a local branch through Git or a [source integration](../integrations/source/_index.md)
+
+-   [branch](../other/glossary.md#branch) existing environments using the [CLI](/administration/cli/_index.md)
+    or the [Console](../administration/web/_index.md)
 
-You can have branches that aren't tied to a running instance of your application.
-These are called [inactive environments](../other/glossary.md#inactive-environment).
+When you branch an environment, you might want to create exact replicas of it.
+In this case, each new environment inherits all of the data and services from its parent environment.
+This includes databases, network storage, queues, and routing configurations.
+
+You can create Platform.sh environments on demand.
+Each environment is tied to a Git branch.
+If you use a [source integration](../integrations/source/_index.md),
```

Comment:
:pick: Since source integrations are linked just before this in this section, it might not be necessary here. :shrug: 
```suggestion
If you use a source integration,
```